cmd/integration/harness.go

```diff
@@ -90,76 +77,28 @@ func NewHarness(cfg *ServerConfig, args []string) (*Harness, error) {
 	return h, nil
 }
 
-func isListening(addr string) bool {
-	conn, _ := net.DialTimeout("tcp", addr, 1*time.Second)
-	if conn != nil {
-		_ = conn.Close()
-		return true
-	}
-	return false
-}
-
-func killProcess(address string) error {
-	addr, err := net.ResolveTCPAddr("tcp", address)
-	if err != nil {
-		return err
-	}
-
-	var cmd *exec.Cmd
-	if runtime.GOOS == "windows" {
-		args := fmt.Sprintf("(Get-NetTCPConnection -LocalPort %d).OwningProcess -Force", addr.Port)
-		cmd = exec.Command("Stop-Process", "-Id", args)
-	} else {
-		args := fmt.Sprintf("lsof -i tcp:%d | grep LISTEN | awk '{print $2}' | xargs kill -9", addr.Port)
-		cmd = exec.Command("bash", "-c", args)
-	}
-
-	var errb bytes.Buffer
-	cmd.Stderr = &errb
-
-	if err := cmd.Start(); err != nil {
-		return err
-	}
-
-	if err := cmd.Wait(); err != nil {
-		return fmt.Errorf("error during killing process: %s | %s", err, errb.String())
-	}
-
-	return nil
-}
-
 func main() {
+	log.JSONLog(true)
 	// os.Args[0] contains the current process path
 	h, err := NewHarnessDefaultServerConfig(os.Args[1:])
```

Comment:
please talk to @gavraz about collecting panics thrown to terminal from the spawned spacemesh process

Reply:
fixed in e5add110af94a4a0352e14bfa2befe99788470a0

Reply:
I did, do you see something wrong here?